pkg/multicluster/target.go

```diff
@@ -111,7 +109,8 @@ func (t *ClusterGatewayTarget) GetName() string {
 }
 
 func (t *ClusterGatewayTarget) GetShortCode() string {
-	return ToBase36hash(t.GetName())
+	log.Log.V(1).Info("getting short code", "clusterName", t.GetName(), "clusterID", common.ToBase36HashLen(t.GetName(), common.ClusterIDLength))
```

Comment:
Do we need this log? Probably better to log where ever it is being used. This might end up elsewhere, library code, possibly in dns operator itself at some point.